templates/search/blocks/search_results_map.html

```diff
@@ -1,11 +1,12 @@
 {% load search_tags %}
-
+<div class="ohos-search-results-map__info">
+    <i class="ohos-search-results-map__info-icon fa-solid fa-circle-info" style="font-size:42px;display:contents;"></i>
```

Comment:
The icon is showing in the middle of text. The descriptions says - Align the ‘i' icon to the top of the text

![image](https://github.com/user-attachments/assets/57bceb6d-0193-4ca6-8bc7-fe57c94f3307)

Reply:
I've now positioned the icon to the top.

Reply:
Thanks @JohnHeeryTNA 

I checked, the icon looks moved to the top in full screen mode

Previous :
![image](https://github.com/user-attachments/assets/54abecc5-c10a-4cfc-89f5-492d66932292)

New changes:
![image](https://github.com/user-attachments/assets/8871f504-658c-440e-bcc8-ed0b6367d51c)